blueprint/index.md

```diff
@@ -175,25 +176,25 @@ To enable communication from Genesys Cloud to Microsoft Azure and Microsoft Team
       **Note:** Click **Yes** in the Publish Action window to publish the data action. The import action modifies only the data action configuration and not the data action contract.
       :::
 
-6.  Return to the web services data action integration and verify that the data action is in **Active** status.
+   6.  Return to the web services data action integration and verify that the data action is in **Active** status.
 
 ### Create a custom role for use with Genesys Cloud OAuth client
 
-1. Navigate to **Roles/Permissions**.
+1. Navigate to **Roles/Permissions** and click **Add Role**.
 
```

Comment:
**Roles** > **Permissions**

Reply:
I have used Roles/Permissions as it displays the same in the UI.